### ISSUE TITLE
Not able to setup the db in production

### ISSUE THREAD

**Issue by anibha88** [InfraexDev/peatio#20]
When I ran 
bundle exec rake db:setup RAILS_ENV=development --trace

Its giving below errors
/home/ubuntu/peatio/current/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 65910 was converted to "65910". (StructuredWarnings::StandardWarning)
** Invoke db:setup (first_time)
** Invoke db:schema:load_if_ruby (first_time)
** Invoke db:create (first_time)
** Invoke db:load_config (first_time)
** Execute db:load_config
** Execute db:create
Couldn't create database for {"adapter"=>"mysql2", "encoding"=>"utf8", "database"=>"peatio_development", "username"=>"root", "password"=>nil, "pool"=>10, "reconnect"=>true}, {:charset=>"utf8", :collation=>"utf8_unicode_ci"}
(If you set the charset manually, make sure you have a matching collation)
Couldn't create database for {"adapter"=>"mysql2", "encoding"=>"utf8", "database"=>"peatio_test", "username"=>"root", "password"=>nil, "pool"=>10, "reconnect"=>true}, {:charset=>"utf8", :collation=>"utf8_unicode_ci"}
(If you set the charset manually, make sure you have a matching collation)
** Invoke environment (first_time)
** Execute environment
rake aborted!
**NameError: uninitialized constant MailsViewer**

Am I missing something? Please help!

**Comment by muhammednagy** [comment]
are you trying to run development env on a production server?
which document did you follow?

**Comment by anibha88** [comment]
Well, Sorry! The issue was basically related to access to the database:
https://stackoverflow.com/questions/21719738/rails-and-mysql2-access-denied-for-user-rootlocalhost-using-password-no

I tried several ways like changing the contents of database.yml and didn't succeed. In this process somewhere I landed with the above error message.

Still, I find my question irrelevant to the issue I am facing. I'll, therefore, close this.

Thanks!
